docs/template_plugin/tests/functional/op_reference/reduction.hpp

```diff
@@ -4,14 +4,11 @@
 
 #include <gtest/gtest.h>
 
-#include <ie_core.hpp>
-#include <ie_ngraph_utils.hpp>
-#include <ngraph/ngraph.hpp>
-#include <shared_test_classes/base/layer_test_utils.hpp>
-#include <vector>
-
 #include "base_reference_test.hpp"
 #include "ngraph_functions/builders.hpp"
+#include "ngraph/shape_util.hpp"
```

Comment:
@ilyachur do we need to move this file to new API? or make it internal for new operations?

Reply:
Why do we need to include this header?
In general I would like to try to use this API internally for new ops.

Reply:
`ngraph::reduce` from this file is used